tests/TestCase.php

```diff
@@ -15,7 +15,8 @@ protected function setUp()
 	{
 	    parent::setUp();
 
-	    putenv('API_HOST=https://api.ethicaljobs.com.au');
+	    putenv('API_HOST=api.ethicaljobs.com.au');
```

Comment:
Is this not a little dangerous to be setting the default endpoint as production within our test-cases? Imagine a developer writes a test  that mutates some data on `api-app:production`. Also I'd vouch for setting envs in the phpunit.xml its more of a standard.

Reply:
Good point, changed to staging.

Reply:
Moving to phpunit.xml doesn't seem to work because it only applies the env vars once at the start, not before each test case.